lib/beaker-pe/install/pe_utils.rb

```diff
@@ -840,7 +840,11 @@ def check_console_status_endpoint(host)
           attempts_limit = options[:pe_console_status_attempts] || 9
           step 'Check Console Status Endpoint' do
             match = repeat_fibonacci_style_for(attempts_limit) do
-              output = on(host, "curl -s -k https://localhost:4433/status/v1/services --cert /etc/puppetlabs/puppet/ssl/certs/#{host}.pem --key /etc/puppetlabs/puppet/ssl/private_keys/#{host}.pem --cacert /etc/puppetlabs/puppet/ssl/certs/ca.pem", :accept_all_exit_codes => true)
+              if (host['pe_ver'] == '2016.1.1')
+                output = on(host, "curl -s -k https://localhost:4433/status/v1/services?level=critical --cert /etc/puppetlabs/puppet/ssl/certs/#{host}.pem --key /etc/puppetlabs/puppet/ssl/private_keys/#{host}.pem --cacert /etc/puppetlabs/puppet/ssl/certs/ca.pem", :accept_all_exit_codes => true)
+              else
+                output = on(host, "curl -s -k https://localhost:4433/status/v1/services --cert /etc/puppetlabs/puppet/ssl/certs/#{host}.pem --key /etc/puppetlabs/puppet/ssl/private_keys/#{host}.pem --cacert /etc/puppetlabs/puppet/ssl/certs/ca.pem", :accept_all_exit_codes => true)
```

Comment:
@shaigy rather than repeating this entire command wholesale, I think it would be nicer if we pointed out just the difference for 2016.1.1 for readability.  So this would look something like this:
```ruby
query_params = host['pe_ver'] == '2016.1.1' ? '?level=critical' : ''
output = on(host, "curl -s .... v1/services#{query_params} --cert ...")
```
(that's assuming that query param is the only change, I just glanced at it)